Explorer/Assets/DCL/Passport/Configuration/PassportAdditionalFieldsValueListConfigurationSO.cs

```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+
+namespace DCL.Passport.Configuration
+{
+    [CreateAssetMenu(fileName = "PassportAdditionalFieldsValueListConfiguration", menuName = "SO/PassportAdditionalFieldsValueListConfiguration")]
+    public class PassportAdditionalFieldsValueListConfigurationSO : ScriptableObject
```

Comment:
There is not a special reason, I prefered to divide it in different assets only in case in the future we need part of these settings in another features (for example the list of countries or whatever).

![image](https://github.com/user-attachments/assets/599a2a6a-3945-42a3-ad58-3412be879c5e)
![image](https://github.com/user-attachments/assets/beabff77-4768-46d0-9447-b3302ebd4f33)